src/libtools/histogram.cc

```diff
@@ -38,8 +38,8 @@ void Histogram::ProcessData(DataCollection<double>::selection *data) {
   _pdf.assign(_options._n, 0);
 
   if (_options._auto_interval) {
-    _min = numeric_limits<double>::max();
-    _max = numeric_limits<double>::min();
+    _min = std::numeric_limits<double>::max();
```

Comment:
Any particular reason why you are using std in the source files, is it not safe to simply do a using namespace std?

Reply:
in general I would not use using namespace anywhere. So when it is just a couple std:: I just add them